script.js

```diff
@@ -0,0 +1,167 @@
+const phones = [
+  {
+    model: "iPhone 13",
+    price: 900,
+    ozu: [
+      {
+        name: "16GB",
+        price: 100,
+      },
+      {
+        name: "32GB",
+        price: 200,
+      },
+      {
+        name: "64GB",
+        price: 500,
+      },
+    ],
+    memory: [
+      {
+        name: "128GB",
+        price: 100,
+      },
+      {
+        name: "256GB",
+        price: 200,
+      },
+      {
+        name: "512GB",
+        price: 325,
+      },
+      {
+        name: "1TB",
+        price: 500,
+      },
+    ],
+  },
+  {
+    model: "Samsung Galaxy A12",
+    price: 400,
+    ozu: [
+      {
+        name: "16GB",
+        price: 40,
+      },
+      {
+        name: "32GB",
+        price: 100,
+      },
+    ],
+    memory: [
+      {
+        name: "128GB",
+        price: 50,
+      },
+      {
+        name: "256GB",
+        price: 150,
+      },
+      {
+        name: "1TB",
+        price: 200,
+      },
+    ],
+  },
+];
+
+const modelSelector = document.querySelector('select[name="model"]');
+const ozuSelector = document.querySelector('select[name="ozu"]');
+const memorySelector = document.querySelector('select[name="memory"]');
+const form = document.getElementById("phone-form");
+const error = document.getElementById("error");
+const result = document.getElementById("result");
+
+modelSelector.onchange = (e) => {
```

Comment:
It's not the best thing to add event listeners, the common way is by `addEventListener`
For change detection you can use the appropriate value
`addEventListener('change', (event) => {});`